sdk/docs/iot/pseudo_code_examples.md

```diff
@@ -0,0 +1,127 @@
+# Azure Embedded C SDK IoT Client Pseudo Code Examples
```

Comment:
The example looks like valid C code (with C specific syntax elements, even if it doesn't compile) rather than language-agnostic pseudo code.

Maybe rename this to something like truncated/abbreviated/condensed//etc.
```suggestion
# Azure Embedded C SDK IoT Client Abridged Code Examples
```

Reply:
@jspaith or @danewalton any thoughts on this?

Reply:
I don't really have strong opinions, though for what its worth when I write something i call pseudo code it comes out as (very bad) C# so this indeed might be more a coding pattern.